automation_tools/__init__.py

```diff
@@ -2127,6 +2130,151 @@ def get_hostname_from_ip(ip, timeout=3):
     return output[ip]
 
 
+def get_rhevm_client():
```

Comment:
The __init__.py file is to big already, what about start a new module for these tasks/functions? Eventually we also would move all the upgrade related tasks to that module.